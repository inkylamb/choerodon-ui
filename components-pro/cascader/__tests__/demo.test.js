import demoTest from '../../../tests/shared/demoTest';

demoTest('email-field-pro');
