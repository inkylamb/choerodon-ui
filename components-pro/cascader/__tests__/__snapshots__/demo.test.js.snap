// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ./components-pro/email-field/demo/basic.md correctly 1`] = `
<div
  class="c7n-row"
  style="margin-left:-5px;margin-right:-5px"
>
  <div
    class="c7n-col-8"
    style="padding-left:5px;padding-right:5px"
  >
    <span
      class="c7n-pro-input-wrapper c7n-pro-input-empty"
    >
      <label>
        <input
          autocomplete="off"
          class="c7n-pro-input"
          placeholder="请输入"
          type="email"
          value=""
        />
      </label>
    </span>
    <span />
  </div>
  <div
    class="c7n-col-8"
    style="padding-left:5px;padding-right:5px"
  >
    <span
      class="c7n-pro-input-wrapper"
    >
      <label>
        <input
          autocomplete="off"
          class="c7n-pro-input"
          placeholder="清除按钮"
          type="email"
          value="点击清除"
        />
        <div
          class="c7n-pro-input-inner-button c7n-pro-input-clear-button"
        >
          <i
            class="icon icon-close"
          />
        </div>
      </label>
    </span>
    <span />
  </div>
  <div
    class="c7n-col-8"
    style="padding-left:5px;padding-right:5px"
  >
    <span
      class="c7n-pro-input-wrapper c7n-pro-input-disabled"
    >
      <label>
        <input
          autocomplete="off"
          class="c7n-pro-input"
          disabled=""
          readonly=""
          type="email"
          value="不可用"
        />
      </label>
    </span>
    <span />
  </div>
</div>
`;

exports[`renders ./components-pro/email-field/demo/controlled.md correctly 1`] = `
Array [
  <span
    class="c7n-pro-input-wrapper"
  >
    <label>
      <input
        autocomplete="off"
        class="c7n-pro-input"
        type="email"
        value="abc@123.com"
      />
    </label>
  </span>,
  <span />,
]
`;

exports[`renders ./components-pro/email-field/demo/dataset.md correctly 1`] = `
Array [
  <span
    class="c7n-pro-input-wrapper c7n-pro-input-required"
  >
    <label>
      <input
        autocomplete="off"
        class="c7n-pro-input"
        name="email"
        type="email"
        value="123@abc.com"
      />
    </label>
  </span>,
  <span />,
]
`;
